Optimize query in get_group_category_info

stsnel opened the pull request:
Using IN instead of wildcard enables use of default iCAT database indexes, which speeds up the query on large environments.

stsnel committed:
Optimize query in get_group_category_info

Using IN instead of wildcard enables use of default iCAT database
indexes, which speeds up the query on large environments.

lwesterhof reviewed (approved):
LGTM, we could make the same optimization in uuGroupGetCategory (uuGroup.r). That could speedup uuGroupModify.

stsnel committed:
uuGroupGetCategory: split query

Split uuGroupGetCategory GenQuery so that it uses default iCAT database
indexes. This improves performance on large environments.
